.azure-pipelines/common-templates/install-tools-template.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+steps:
+    - task: UseDotNet@2
+      displayName: 'Use .NET Core SDK 2.x'
+      inputs:
+        debugMode: false
+        version: 2.x
+
+    - task: UseDotNet@2
+      displayName: 'Use .NET Core SDK 3.x'
+      inputs:
+        debugMode: false
+        version: 3.x
+
+    - task: UseDotNet@2
+      displayName: 'Use .NET Core SDK 5.x'
+      inputs:
+        debugMode: false
+        version: 5.x
```

Comment:
Do we need all these versions of .NET Core? Can we settle on just one version to help isolate .NET related issues in the future?

Reply:
It's also worth considering the End of Life (EOL) for some of these versions. See https://dotnet.microsoft.com/platform/support/policy/dotnet-core.

Reply:
Some of the security tools require those older versions of dotnet. 
Also its specified as `2.x` which picks the appropriate version.

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops#task-inputs
2.x => Install latest in major version.